pyproject.toml

```diff
@@ -21,6 +21,7 @@ python-i18n = "^0.3"
 pyyaml = "^5.3"
 distro = "^1.5"
 packaging = "^20.4"
+docker-compose = "^1.29.1"
```

Comment:
Could this version ever be incompatible with a system-installed Docker?

Reply:
My understanding is that Compose's compatibility with Docker is decided by the version of the docker-compose.yaml spec we use, not the version of the command line utility. We will have to be mindful when we upgrade that version, but it doesn't look like we have to worry about this version.

See: https://docs.docker.com/compose/compose-file/compose-versioning/